templates/index.html

```diff
@@ -38,390 +41,523 @@ <h2 class="p-heading--two">Say hello to your physical&nbsp;cloud</h2>
   </div>
 </section>
 
-<section class="p-strip p-strip--light is-deep">
+<section class="p-strip--light">
   <div class="row">
-    <div class="col-5">
-      <h2 class="u-no-margin--bottom">The smartest way to manage bare metal</h2>
+    <h3 class="p-muted-heading u-align-text--center">A selection of customers</h3>
+    <ul class="u-no-margin">
```

Comment:
I am in favour of using snapcrafts fluid logo grid pattern here.

Reply:
Included now